microcloud/test/includes/microcloud.sh

```diff
@@ -74,6 +77,35 @@ EOF
 )
 fi
 
+if [ -z "${IGNORE_CEPH_NETWORKING}" ]; then
+  if [ -n "${CEPH_PUBLIC_NETWORK}" ]; then
+    setup=$(cat << EOF
+${setup}
+${CEPH_PUBLIC_NETWORK}                                  # setup public network
+EOF
+)
```

Comment:
Would you mind using the following equivalent syntax?
```suggestion
    setup="${setup}
${CEPH_PUBLIC_NETWORK}"                                 # setup public network
)
```

I'd put a PR doing the same for the existing code. The reason for this style change is that it produces less debug (`set -x`) noise:

```
$ sh -x
$ setup=""
+ setup=
$ setup="${setup}
> foo"
+ setup=
foo
$ setup="${setup}
> bar"
+ setup=
foo
bar
$ 
```

Versus:

```
$ sh -x
$ setup=""   
+ setup=
$ setup=$(cat << EOF
> ${setup}
> foo
> EOF
> )
+ cat
+ setup=
foo
$ setup=$(cat << EOF
> ${setup}
> bar
> EOF
> )
+ cat
+ setup=
foo
bar
$ 
```
```

Reply:
fixed